### PR TITLE
Visually indicate which tab a user is on

### PR DESCRIPTION
**Pull request by NavyAdmiral**
For [#165974161](https://www.pivotaltracker.com/story/show/165974161)

Perhaps it's better to wait for the official design, but gave it a shot anyway

asd

I checked the Nav on twitter and they do the same thing;
Other ideas: italics, font-weight, color manipulation, ...

### CODE REVIEW

File: operator_ui/src/theme.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default {
     primary: {
       light: '#E5F1FF',
       main: '#3c40c6',
+      secondary: 'rgba(	60, 64, 198, 0.75)',
```
Comment (rupurt) [inline operator_ui/src/theme.js]: ✂️ spacing

**Comment by NavyAdmiral** [comment]
@rupurt My bad, I read the story wrong. Since the approaches taken were the same for all 3 navigations, I decided to commit the changes in this PR. I also noticed that for jobspecs, the url for definition is `/definition`, while for runs it is `/json`. Do we want to be consistent with this?

**Comment by NavyAdmiral** [comment]
@thoughtmerchant Would be nice to hear from you here too. Here's a current preview
asd

I have 2 concerns, is it a good choice to indicate the active tab by leaving the hover effect on? The main problem I have with this is that once a user is on active tab, hovering over other tabs creates a confusing experience. I decided to use the same color with a lower opacity to solve this. FWIW, I've checked how Twitter et al handle this, and they don't even point out the difference between the current active tab, and the one that is being hovered. Attaching gif again for clarity:
dsa

File: operator_ui/src/components/JobRuns/RegionalNav.js [hunk 2]
```diff
@@ -92,8 +104,12 @@ RegionalNav.propTypes = {
   jobRun: PropTypes.object
 }
 
+const mapStateToProps = state => ({
+  url: state.notifications
```
Comment (rupurt) [inline operator_ui/src/components/JobRuns/RegionalNav.js]: I feel like this should be `url: state.notifications.currentUrl`

File: operator_ui/src/containers/Header.js [hunk 3]
```diff
@@ -193,7 +203,8 @@ Header.propTypes = {
 
 const mapStateToProps = state => ({
   authenticated: state.authentication.allowed,
-  fetchCount: fetchCountSelector(state)
+  fetchCount: fetchCountSelector(state),
+  url: state.notifications
```
Comment (rupurt) [inline operator_ui/src/containers/Header.js]: `url: state.notifications.currentUrl`

**Comment by rupurt** [comment]
@NavyAdmiral from memory the original tab was called `Definition` on `JobSpec#Show`. I'm happy to change the route to `/json` to make it consistent.

File: operator_ui/src/Private.js [hunk 4]
```diff
@@ -109,7 +109,7 @@ class Private extends React.Component {
                 />
                 <PrivateRoute
                   exact
-                  path="/jobs/:jobSpecId/definition"
+                  path="/jobs/:jobSpecId/json"
```
Comment (rupurt) [inline operator_ui/src/Private.js]: This also needs to `mv operator_ui/src/pages/jobs/_jobSpecId_/definition.js` -> `operator_ui/src/pages/jobs/_jobSpecId_/json.js`

**Comment by NavyAdmiral** [comment]
@rupurt I feel like there's a room to extract code from the several RegionalNav's, CodeClimate complains about it too. How do you feel about it? Should that be done in further PR's separate from this PR?

**Pull request merged**